src/ol_infrastructure/applications/edxapp/Pulumi.applications.edxapp.mitxonline.Production.yaml

```
@@ -9,7 +9,7 @@ config:
   consul:scheme: https
   edxapp:waffle_flags:
   - ["content_tagging.disabled", "--create", "--everyone"]
-  - ["contentstore.new_studio_mfe.use_new_video_uploads_page", "--create", "--everyone"]
+  - ["contentstore.new_studio_mfe.use_new_video_uploads_page", "--create", "--disabled"]
```

```suggestion
  - ["contentstore.new_studio_mfe.use_new_video_uploads_page", "--create", "--deactivate"]
```
I was wrong on the syntax